package-lock.json

```diff
@@ -0,0 +1,5389 @@
+{
```

Comment:
We shouldn't have this file committed anymore. We should always use `yarn` which uses `yarn.lock` instead of `package-lock.json`.